src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2185,14 +2189,21 @@ namespace winrt::TerminalApp::implementation
             {
                 co_await winrt::resume_foreground(Dispatcher());
 
+                if (warnMultiLine)
+                {
+                    const auto focusedTab = _GetFocusedTabImpl();
+                    // Do not warn about multi line pasting if the current tab has bracketd paste enabled.
+                    warnMultiLine &= !focusedTab->GetActiveTerminalControl().BracketedPasteEnabled();
+                }
+
                 // We have to initialize the dialog here to be able to change the text of the text block within it
                 FindName(L"MultiLinePasteDialog").try_as<WUX::Controls::ContentDialog>();
                 ClipboardText().Text(text);
 
                 // The vertical offset on the scrollbar does not reset automatically, so reset it manually
                 ClipboardContentScrollViewer().ScrollToVerticalOffset(0);
 
-                ContentDialogResult warningResult;
+                ContentDialogResult warningResult = ContentDialogResult::Primary;
```

Comment:
Curious, what was the default-constructed value for this `ContentDialogResult`. Did setting it to `Primary` change anything?

Reply:
The default-constructed value is garbage data, like an uninitialized integer. I had to make it "Primary" to make it not `co_return` directly. *read the logic*

Reply:
Indeed! It was uninitialized!

If you initialize it with `{}`, it will be whichever one matches 0